docroot/profiles/custom/cgov_site/modules/custom/cgov_syndication/cgov_syndication.module

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations for content syndication.
+ */
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Implements hook_preprocess_html().
+ */
+function cgov_syndication_preprocess_html(&$variables) {
+
+  // Add the `meta` information we need for syndicated content.
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if (!empty($node) && $node instanceof Node) {
+    if ($node->hasField('field_hhs_syndication')) {
+      $canonical = $node->toUrl('canonical')->toString();
+      $variables['syndication_canonical'] = "https://www.cancer.gov$canonical";
```

Comment:
Where do we store this site-specific configuration? In the previous multisite setup we used to have for the Drupal 7 sites (EBMS, MyNCI, etc.) that typically went into the `settings.php` files, which were maintained by hand, separately from version control. I assume we don't want to use that approach. We're only going to have one profile, so it can't go there (and probably couldn't go there even if we had multiple profiles, as I assume a profile could be used by more than one site). It can't be the `cgov_core` module (despite the implication that might be conveyed by its name), as its `README.md` says it's used by "any web site."